src/ParallelAlgorithms/Interpolation/InterpolationTargetDetail.hpp

```diff
@@ -468,27 +468,28 @@ void add_received_variables(
 ///
 /// block_logical_coords is called by an Action of InterpolationTarget.
 ///
-/// Currently two Actions call block_logical_coords:
-/// - SendPointsToInterpolator (called by AddTemporalIdsToInterpolationTarget
-///                             and by FindApparentHorizon)
-/// - InterpolationTargetVarsFromElement (called by DgElementArray)
+/// Currently one Action directly calls this version of block_logical_coords:
 /// - InterpolationTargetSendTimeIndepPointsToElements
 ///   (in InterpolationTarget ActionList)
-template <typename InterpolationTargetTag, typename DbTags,
-          typename Metavariables, typename TemporalId>
-auto block_logical_coords(const db::DataBox<DbTags>& box,
-                          Parallel::GlobalCache<Metavariables>& cache,
-                          const TemporalId& temporal_id) {
+/// and one Action indirectly calls this version of block_logical_coords:
+/// - SendPointsToInterpolator (called by AddTemporalIdsToInterpolationTarget
+///                             and by FindApparentHorizon)
+template <typename InterpolationTargetTag, typename Metavariables,
+          typename TemporalId>
+auto block_logical_coords(
+    Parallel::GlobalCache<Metavariables>& cache,
```

Comment:
I'd expect the `cache` can be constant. It only needs to be non-const if being used for communication